src/reactviews/pages/ConnectionDialog/connectionPageContainer.tsx

```diff
@@ -51,7 +51,7 @@ export const ConnectionInfoFormContainer = () => {
         <div className={formStyles.formRoot}>
             <ConnectionHeader />
 
-            <div className={formStyles.formDiv}>
+            <div className={formStyles.formDiv} style={{ overflow: "auto" }}>
```

Comment:
I put it here since it's overriding the default style for `formDiv` and it's only being used once, so I didn't think it needed to be a new class, but if you feel strongly here, I'm happy to move it.

Reply:
Got it. I think it's fine to keep it as is since it's only being used once.